jenkins-scripts/dsl/ignition.dsl

```diff
@@ -293,15 +293,8 @@ ignition_software.each { ign_sw ->
                 wget https://raw.githubusercontent.com/osrf/bash-yaml/master/yaml.sh -O yaml.sh
                 source yaml.sh
```

Comment:
I think these 2 lines could be deleted as well (`wget` and `source`)

Reply:
✂️✂️✂️ 441dfbf